### ISSUE TITLE
create:comments outputs "Call to undefined method Drupal\comment\Entity\Comment::getTitle()"

### ISSUE THREAD

**Issue by sikko** [hechoendrupal/drupal-console#3628]
### How to reproduce
- Create some nodes
- Create some users
- Launch the create:comments command and answer the prompted questions
- Press enter to execute the command

**Setup**
- Drupal version: 8.4.3
- Console version: 1.3.1
- Console Launcher version: 1.3.1

**Comment by LOBsTerr** [comment]
A small addition it happens, when the script face some issues during the creation of the comment and it throws an exception. For example we can pass invalid node id - 12121212212121212121

**Comment by LOBsTerr** [comment]
PR https://github.com/hechoendrupal/drupal-console/pull/3669
1) The problem of the warning was, that the wrong method getTitle was used. Comment entity have getSubject method. Probably, code was copied from "Create node" command.
2) I noticed that there is no validation for node id, I have added it.
3) Also, I improved a little bit the structure of the code.

I have a question, how can I add the translated string for Node id validation?
Should I create a separate PR for drupal-console-en ?
